lab4/oolab/src/test/java/agh/ics/oop/AnimalTest.java

```diff
@@ -0,0 +1,126 @@
+//package agh.ics.oop;
```

Comment:
co do zasady nie powinniśmy komentować testów, które przestają działać - to normalne że apka się rozwija i powinniśmy też zmieniać/poprawiać stale testy ;)